mne/viz/_3d.py

```diff
@@ -1143,3 +1143,118 @@ def plot_dipole_locations(dipoles, trans, subject, subjects_dir=None,
         fig.scene.x_plus_view()
 
     return fig
+
+
+def snapshot_brain_montage(fig, montage, hide_sensors=True):
+    """Take a snapshot of a Mayavi Scene and project channels onto 2d coords.
+
+    Note that this will take the raw values for 3d coordinates of each channel,
+    without applying any transforms.
+
+    Parameters
+    ----------
+    fig : instance of Mayavi Scene
+        The figure on which you've plotted electrodes using `plot_trans`.
+    montage : instance of DigMontage | instance of `Info`
+        The digital montage for the electrodes plotted in the scene. If `Info`,
+        channel positions will be pulled from the `loc` field of `chs`.
+    hide_sensors : bool
+        Whether to remove the spheres in the scene before taking a snapshot.
+
+    Returns
+    -------
+    xy : array, shape (n_channels, 2)
+        The 2d location of each channel on the image of the current scene view.
+    im : array, shape (m, n, 3)
+        The screenshot of the current scene view
+    """
+    from mayavi import mlab
+    from ..channels import Montage, DigMontage
+    from .. import Info
+    if isinstance(montage, (Montage, DigMontage)):
+        xyz = np.vstack(montage.dig_ch_pos.values())
+        ch_names = montage.dig_ch_pos.keys()
+    elif isinstance(montage, Info):
+        xyz = np.vstack([ich['loc'][:3] for ich in montage['chs']])
+        ch_names = np.array([ich['ch_name'] for ich in montage['chs']])
+
+    xy = _3d_to_2d(fig, xyz)
+    xy = dict(zip(ch_names, xy))
+    pts = fig.children[-1]
+    if hide_sensors is True:
+        pts.visible = False
+
+    # Take screenshot and flip up/dn
+    im = mlab.screenshot(fig)[::-1, ...]
```

Comment:
If this really works for you, perhaps there should be a param for ``invert_y``. For me (mayavi 4.4.3), this flips the brain upside down.